Spanner/src/Operation.php

```diff
@@ -119,10 +119,20 @@ public function deleteMutation($table, KeySet $keySet)
      *     Configuration options.
      *
      *     @type string $transactionId The ID of the transaction.
+     *     @type bool $returnCommitStats If true, return the full response.
+     *           **Defaults to** `false`.
      * }
      * @return Timestamp The commit Timestamp.
      */
     public function commit(Session $session, array $mutations, array $options = [])
+    {
+        return $this->commitWithResponse($session, $mutations, $options)[0];
+    }
+
+    /**
+     * @internal
```

Comment:
I apologize, I wasn't clear earlier, `@internal` is good, but we should still have the remaining docblock data for args and return type.

Reply:
Docblock added :+1:

Reply:
Docblock added :+1: